server/app/services/export/ExporterService.java

```diff
@@ -127,49 +127,49 @@ private CsvExportConfig generateDefaultCsvExportConfig(
    */
   public String getProgramCsv(long programId) throws ProgramNotFoundException {
     ProgramDefinition program = programService.getProgramDefinition(programId);
-    CsvExporter csvExporter;
-    if (program.exportDefinitions().stream()
-        .anyMatch(exportDefinition -> exportDefinition.csvConfig().isPresent())) {
-      csvExporter = exporterFactory.csvExporter(program.toProgram());
-    } else {
-      csvExporter = exporterFactory.csvExporter(generateDefaultCsvConfig(programId));
-    }
+    Optional<CsvExportConfig> maybeExportConfig =
+        program.exportDefinitions().stream()
+            .filter(exportDefinition -> exportDefinition.csvConfig().isPresent())
+            .map(exportDefinition -> exportDefinition.csvConfig().get())
+            .findFirst();
```

Comment:
suggestion (non-blocking): i'm pretty sure that admin-specified, per-program export configuration was a planned feature that never got implemented. could remove this logic altogether if we wanted i think, though that may be refactor outside the scope of this PR